### DISCUSSION TITLE
Deleting VulnerabilityReports

### DISCUSSION THREAD

**Discussion by be-lka** [aquasecurity/trivy-operator#1217] [Help & Support] [answered]
Hi all! 
I'm experimenting with the Trivy Operator for the first time and running it in a sandbox cluster. Only after I've installed it using Helm is when I've noticed the possibility of customizing the `severity` attribute of the scanner to only see HIGH and CRITICAL issues, as we're interested in doing. 
Now, I have lots of Vulnerability Reports with LOW, MEDIUM and INFORMATIONAL issues in my cluster.
I'd like to delete this reports and re-run the Operator, to see only HIGH and CRITICAL issues. 
Is this possible? 
I haven't found anything related to deleting the reports in the official documentation. 
Thanks in advance!

**Comment by chen-keinan** [comment]
sure , see uninstall section in  [docs](https://aquasecurity.github.io/trivy-operator/v0.13.2/getting-started/installation/helm/)

**Reply by be-lka** [reply]
Hi @chen-keinan! Just to make sure - can I just delete the CRD of the Vulnerability Report or I must uninstall the Operator completely and install it back again?

**Reply by chen-keinan** [reply]
you heed to do `helm uninstall` followed by `delete crds`. 
 afterwards  your env. will be clean

**Reply by be-lka** [reply]
Isn't there a way to delete just the Vulnerability Reports without having to uninstall the Trivy Operator? All I want is to clean the current reports and let the Operator that is already working re-generate them with the customized Severity.

**Reply by chen-keinan** [reply, marked as answer]
you can delete it via `kubectl delete ...` as any resource

**Reply by be-lka** [reply]
Got it. Thank you!
```
❯ kubectl delete vulnerabilityreport --all-namespaces --all
vulnerabilityreport.aquasecurity.github.io "cronjob-daily" deleted
vulnerabilityreport.aquasecurity.github.io "cronjob-hourly" deleted
vulnerabilityreport.aquasecurity.github.io "replicaset-api" deleted
.
.
.
```
